### PR TITLE
Feature/8/for loop

### PR DESCRIPTION
**Pull request by judit-nahaj**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,11 +7,11 @@
...
-
+- Added predefined precompiler: forLoop ([#8](https://github.com/judit-nahaj/gherkin-precompiler/issues/8))
 
 ## 1.0.0 - 2017-09-25
 
 ### Added
 
 - Add code to apply preprocessing rules to Gherkin feature files ([#1](https://github.com/judit-nahaj/gherkin-precompiler/issues/1))
-- Add predefined precompilers: replacer ([#2](https://github.com/judit-nahaj/gherkin-precompiler/issues/2))
+- Add predefined prenpm logicompilers: replacer ([#2](https://github.com/judit-nahaj/gherkin-precompiler/issues/2))
```
Comment (szikszail) [inline CHANGELOG.md]: Could you fix this again? :)

File: lib/builtIn/ForLoop.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    format: '${name} (${i})'
+};
+
+/**
+ * ForLoop to repeat a scenario for a given number of times.
+ * @class
+ * @extends DefaultConfig
+ */
+class ForLoop extends DefaultConfig {
+    //TODO: constructor config scenario outline numberingből
```
Comment (szikszail) [inline lib/builtIn/ForLoop.js]: Could you please remove TODOs if they are not necessary anymore?

File: lib/builtIn/ForLoop.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+const DefaultConfig = require('../DefaultConfig');
+const DEFAULT_CONFIG = {
```
Comment (szikszail) [inline lib/builtIn/ForLoop.js]: Could you please define this as a type called `ForLoopConfiguration` and use it in the documentation of this class?

File: lib/builtIn/ForLoop.js (hunk 2 above)
Comment (judit-nahaj) [reply lib/builtIn/ForLoop.js]: done

File: CHANGELOG.md (hunk 1 above)
Comment (judit-nahaj) [reply CHANGELOG.md]: done :D

File: lib/builtIn/ForLoop.js (hunk 3 above)
Comment (judit-nahaj) [reply lib/builtIn/ForLoop.js]: fixed, i hope, can you check again please?

**Pull request merged**